README.md

```diff
@@ -27,6 +27,19 @@ style GW fill:#81c784
 style C fill:#e1f5fe
 ```
 
+### Stream Management
+
+Streams are automatically tracked and cleaned up after a certain amount of inactivity.
+This prevents memory leaks from:
+
+- Crashed or disconnected publishers
+- Streams that never reach Phase::End
```

Comment:
what is Phase::End? is that a special message/signal given by one of the clients? publisher or consumer?